roles/download/defaults/main.yml

```diff
@@ -435,6 +437,17 @@ helm_archive_checksums:
     v3.5.1: d0ada80576f8016d1cc38a06d225a4379a53e88e3e26b417e6de5db05a090ce4
     v3.5.2: 126a72e2b209194fd2735861f0cf8bd5b5d1386eedd6aed6e0e050dca80d493e
 
+crun_checksums:
+  arm: 0
+  amd64:
+    0.16: a16508a9c15a2aa898d6ba18bbc394cd37cdb4e3968f177f2fbb7b70a8a0f4fb
+    0.17: af99d543a41c5ef441e9e653b60392e8d6988a56762819a6959031e3154e94c1
+    0.18: e94578c013eae98b0a50477f6bc77963a7c85145bf280da39f9855d69d9cab53
+  arm64:
+    0.16: 0
+    0.17: 0
```

Comment:
Does this `0` mean Kubespray doesn't support 0.16 and 0.17 for arm64?

Reply:
Arm build for older versions are unavailable for [crun](https://github.com/containers/crun/releases).